src/components/analyses/listing/Listing.js

```diff
@@ -277,21 +277,11 @@ function Listing(props) {
         submitAnalysisSupportRequest,
         {
             onSuccess: (responses) => {
-                const shareError =
-                    responses?.length > 1 &&
-                    responses[0].sharing.find((share) =>
-                        share.analyses.find((analysis) => !analysis.success)
-                    );
-
-                if (shareError) {
-                    showErrorAnnouncer(t("statusHelpShareError"));
-                } else {
-                    setShareWithSupportAnalysis(null);
-                    announce({
-                        text: t("statusHelpShareSuccess"),
-                        variant: AnnouncerConstants.SUCCESS,
-                    });
-                }
```

Comment:
In this case, the only response bodies that are being processed are the ones from the analysis sharing and unsharing endpoints. These endpoints have been updated so that if any analysis would fail validation, then the endpoint will return a 400 status, so there's no need to check the response body at all.